lite/Changelog

```diff
@@ -1,6 +1,38 @@
 Stable versions
 ---------------
 
+4.5.0 ():
+	- make xmp_set_position() consistently clear pattern break/jump vars
+	- fix S3M ADPCM4 samples
```

Comment:
This one also appears to be non-applicable to libxmp-lite: the ADPCM sample loader is disabled when `LIBXMP_CORE_PLAYER` is defined: https://github.com/libxmp/libxmp/blob/master/src/loaders/sample.c#L283.

Reply:
Actually, why is the ADPCM loader disabled for libxmp-lite? It looks like a fairly small amount of code.